frontend/src/components/config/App.tsx

```diff
@@ -28,6 +29,8 @@ import Loading from '../core/Loading';
 
 // Set up Google Analytics
 ReactGA.initialize('UA-192641172-2');
+// Set up Hotjar
+hotjar.initialize(2398506, 6);
```

Comment:
On our Hotjar account, it is recording sessions (interesting to look at), but I don't see heatmaps - I think I remember you mentioned that you weren't sure why, do you know if more data is needed there or not sure?

Reply:
I saw the same issue and [found this](https://help.hotjar.com/hc/en-us/articles/360047604633-Why-aren-t-clicks-being-recorded), so I think it has to due with the dynamic HTML. However there's also this option on the Hotjar admin page: 

![Option to fetch new screenshot of sorts](https://user-images.githubusercontent.com/24768574/118755820-710ae080-b81e-11eb-9a55-c8449daf52e4.png)

After doing this it worked for the home page. No clicks showed up for the other pages though, but if you go to the "Move" tab, then it does show a heatmap, so either there's still something wrong with the clicks or we simply don't have enough traffic with people clicking things?

Reply:
Ok gotcha - either way no problem, I don't think this is a priority, thanks for troubleshooting the issue anyway 👍